### PR TITLE
DM-37252: Move util from RubinTV to summit_utils

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: python/lsst/summit/utils/utils.py [hunk 1]
```diff
@@ -601,3 +606,46 @@ def getExpPositionOffset(exp1, exp2, useWcs=True, allowDifferentPlateScales=Fals
                           )
 
     return ret
+
+
+def starTrackerFileToExposure(filename, logger=None):
```
Comment (isullivan) [inline python/lsst/summit/utils/utils.py:611]: This should probably be added to __all__

**Review by isullivan — APPROVED** [review]
Main suggestion is to update `__all__`

File: python/lsst/summit/utils/utils.py (hunk 1 above)
Comment (mfisherlevine) [reply python/lsst/summit/utils/utils.py:611]: Good shout, yes, thanks 🙂

File: python/lsst/summit/utils/utils.py [hunk 2]
```diff
@@ -601,3 +606,46 @@ def getExpPositionOffset(exp1, exp2, useWcs=True, allowDifferentPlateScales=Fals
...
+    logger : `logging.Logger`, optional
+        The logger to use for errors, created if not supplied.
+
+    Returns
+    -------
+    exp : `lsst.afw.image.Exposure`
+        The exposure.
+    """
+    if not logger:
+        logger = logging.getLogger(__name__)
```
[the remark this answers is not in the document]
Comment (mfisherlevine) [reply python/lsst/summit/utils/utils.py:627]: I'm pretty sure that's _always_ defined. Like, just open a python terminal and type `print(__name__)` and it should tell you that you're in `__main__`.

**Pull request merged**